mxcube3/routes/Queue.py

```diff
@@ -939,13 +939,15 @@ def serialize_queue_to_json():
             if child['py/object'].split('.')[1] == 'TaskGroup': #keep going down
                 for grandChild in child['_children']:
                     if grandChild['py/object'].split('.')[1] == 'TaskGroup': #keep going down one more time for the Char
+                        """ the following loop is not needed
```

Comment:
I started this pr before your big refactoring, so looking commit by commit you could see that, but it was not in the final code.